package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@wpsyntex/polylang",
+  "version": "3.1.0",
+  "description": "Adds multilingual capability to WordPress",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "npm run clean && webpack --mode production",
+    "build:dev": "npm run clean && webpack --mode development",
+    "dev": "npm run clean && webpack --mode development --watch",
+    "clean": "rimraf public/css public/js public/",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/polylang/polylang.git"
+  },
+  "keywords": [
+    "multilingual",
+    "bilingual",
+    "translate",
+    "translation",
+    "language",
+    "multilanguage",
+    "international",
+    "localization"
+  ],
+  "author": "WP Syntex",
+  "license": "GPL-3.0+",
+  "bugs": {
+    "url": "https://github.com/polylang/polylang/issues"
+  },
+  "homepage": "https://github.com/polylang/polylang#readme",
```

Comment:
```suggestion
  "homepage": "https://github.com/polylang/legacy-features-for-fse#readme",
```

Reply:
Done in 9991b959c14ff770602a359843938343a099867e